### ISSUE TITLE
Request to use makeItCraft in another project

### ISSUE THREAD

**Issue by nickautomatic** [mattstauffer/makeItCraft#3]
Hi Matt,

I work on an open source project called [Crafty Vagrant](https://github.com/niceandserious/crafty-vagrant) - a Vagrant-based development environment we use at [Nice and Serious](http://niceandserious.com/) to get Craft-based projects up and running quickly.

Currently the install process still involves manually downloading Craft, but your "makeItCraft" script (or a variant thereof) would allow us to simplify the install process to a single step, so I was wondering if you'd mind if we were to include it (or a tweaked version of it) in our project. We'd be very grateful, and would of course credit you in the project README. What do you think?

Many thanks,

Nick

**Comment by mattstauffer** [comment]
@nickautomatic Sounds good to me! :) Please let me know how I can help. And if you do need any changes made, I'd likely just want to just bring them into makeItCraft if possible.

**Comment by nickautomatic** [comment]
@mattstauffer Superb, thanks very much! If you're interested, I've now pushed out [the updated version](https://github.com/niceandserious/crafty-vagrant/commit/1a417ac8e19af5a680e9b8561df5596883b24206) that uses makeItCraft. The changes I've made are that I'm only using the first half of the script (the bit that installs Craft - we don't need dotenv or Composer for the moment), and I've changed the location it unzips to - nothing especially useful, I'm afraid, but feel free to have a rummage and see what I've done. Thanks again. :)
